From: Andrew Morton <akpm@linux-foundation.org>
Subject: zsmalloc-zs_object_copy-replace-email-link-to-doc-checkpatch-fixes
Date: Wed Aug 17 10:42:29 AM PDT 2022


ERROR: code indent should use tabs where possible
#26: FILE: mm/zsmalloc.c:1562:
+         * kunmap_atomic(d_addr). For more details see$

ERROR: code indent should use tabs where possible
#27: FILE: mm/zsmalloc.c:1563:
+         * Documentation/mm/highmem$

total: 2 errors, 0 warnings, 10 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

NOTE: Whitespace errors detected.
      You may wish to use scripts/cleanpatch or scripts/cleanfile

./patches/zsmalloc-zs_object_copy-replace-email-link-to-doc.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Alexey Romanov <avromanov@sberdevices.ru>
Cc: Minchan Kim <minchan@kernel.org>
Cc: Nitin Gupta <ngupta@vflare.org>
Cc: Sergey Senozhatsky <senozhatsky@chromium.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/zsmalloc.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/mm/zsmalloc.c~zsmalloc-zs_object_copy-replace-email-link-to-doc-checkpatch-fixes
+++ a/mm/zsmalloc.c
@@ -1559,8 +1559,8 @@ static void zs_object_copy(struct size_c
 		 * Calling kunmap_atomic(d_addr) is necessary. kunmap_atomic()
 		 * calls must occurs in reverse order of calls to kmap_atomic().
 		 * So, to call kunmap_atomic(s_addr) we should first call
-         * kunmap_atomic(d_addr). For more details see
-         * Documentation/mm/highmem
+		 * kunmap_atomic(d_addr). For more details see
+		 * Documentation/mm/highmem.
 		 */
 		if (s_off >= PAGE_SIZE) {
 			kunmap_atomic(d_addr);
_
